GenderPayGap.WebUI/Controllers/Account/AccountCreationController.cs

```diff
@@ -122,16 +122,27 @@ public IActionResult CreateUserAccountPost(CreateUserAccountViewModel viewModel)
                         + " address");
                 }
             }
+            User user = null;
 
             if (viewModel.HasAnyErrors())
             {
                 return View("CreateUserAccount", viewModel);
             }
 
-            User newUser = CreateNewUser(viewModel);
-            dataRepository.Insert(newUser);
+            // Check for retired account with matching email address
+            User retiredUser = userRepository.FindByEmail(viewModel.EmailAddress, UserStatuses.Retired);
```

Comment:
This looks up the email address and does FirstOrDefault() so will not error if there are multiple matches, although does mean that additional matches might not be removed/ reactivated. The clean up job should resolve this issue though, as it will remove duplicated retired accounts.